app/controllers/tasks/collection_objects/area_and_date_controller.rb

```diff
@@ -0,0 +1,137 @@
+class Tasks::CollectionObjects::AreaAndDateController < ApplicationController
+  include TaskControllerConfiguration
+
+  # GET
+  def index
+    @geographic_areas         = GeographicArea.where('false')
+    @collection_objects       = CollectionObject.where('false')
+    @collection_objects_count = 0
+    @early_date               = CollectionObject.where(project: sessions_current_project_id).order(:created_at).limit(1).pluck(:created_at).first
+    if @early_date.blank?
+      @early_date = Date.parse('1700/01/01')
+    end
+    @late_date = CollectionObject.where(project: sessions_current_project_id).order(created_at: :desc).limit(1).pluck(:created_at).first
```

Comment:
This will be overtaken by changes in the way dates of collection objects (i.e., from CollectingEvent) will be determined. It will be moved to CollectingEvent itself.